### ISSUE TITLE
Dart client should accept/prefer "local" as subdomain for local-cli development now that localhost is deprecated

### ISSUE THREAD

**Issue by spakanati** [nhost/nhost-dart#107]
Currently you have to pass "localhost" or "localhost:1337" as the subdomain for local cli dev:
https://github.com/nhost/nhost-dart/blob/main/packages/nhost_dart/lib/nhost_dart.dart#L50

However, Nhost has deprecated localhost as the subdomain and it seems "local" should be used instead.

**Comment by mhadaily** [comment]
Unfortunately, if that is the case (pleases share the doc thank you :D ) it's not yet supported, and you still need to pass `localhost`. 

I will see if I can spend sometime to fix this.

**Comment by mhadaily** [comment]
How about using serviceUrls directly for local development?

```
      serviceUrls: ServiceUrls(
        authUrl: '',
        storageUrl: '',
        functionsUrl: '',
        graphqlUrl: '',
      ),
    );```

**Comment by vipafattal** [comment]
Any updates, I need to test locally and it's seems there is no way to do that

**Comment by dbarrosop** [comment]
what is the issue? Could someone elaborate a bit more and share logs and errors? I just tested myself and I had no problems using subscriptions using both local.graphql.nhost.run/v1 and local.hasura.nhost.run/v1/graphql

**Comment by dbarrosop** [comment]
Solved in #119
